config/rbac/role.yaml

```diff
@@ -31,3 +31,15 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
This is a hard one, bcs it allows cluster-wide privilege escalation. Wdyt about better docs or a custom target that includes this as a patch rather than a default?

Reply:
I also struggled with this question. I think a middle ground might be to parametrize the roles and lock it behind an `enable: true` option. The documentation would be part of the `values.yaml` so that you don't have to seek it out as a special case.

Reply:
Sounds fine to me!

Reply:
@lhriley are you planning to work on it? Otherwise I'd close the PR

Reply:
I had planned to, but I don't have the bandwidth at the moment.

Reply:
I'm happy to accept a helm chart as additional deployment method!